package.json

```diff
@@ -15,13 +15,16 @@
     "redis": "service redis-server start || redis-server /usr/local/etc/redis.conf || true",
     "devsetup": "cd contentcuration && python manage.py setup --settings=contentcuration.dev_settings",
     "services": "npm-run-all -c --parallel --silent minio redis postgres",
-    "unittests": "yarn run build && yarn run pytest",
+    "unittests": "yarn run build && yarn run pytest && yarn run test-jest",
     "unittests:reusedb": "npm-run-all --parallel --race services pytest:reusedb",
     "apptests": "npm-run-all --parallel --race services cypress:test",
     "test": "npm-run-all unittests apptests",
     "build:dev": "webpack --config webpack.config.js --watch --progress",
+    "test-jest:dev": "yarn run test-jest --watch",
+    "test-jest": "jest --config jest_config/jest.conf.js",
+    "test-jest-cov": "jest --config jest_config/jest.conf.js --coverage",
     "minio:test": "rm -rf ~/.minio_test && minio server ~/.minio_test/ || true",
-    "minio": "minio server ~/.minio_data/ || true",
+    "minio": "MINIO_ACCESS_KEY=development MINIO_SECRET_KEY=development minio server ~/.minio_data/ || true",
```

Comment:
can you explain this change?

Reply:
The server didn't run without it! These are the develop settings for these things in Minio. @aronasorman recommended this change to fix a startup issue I was having.

Reply:
I see these keys in ~/.minio/config.json, so it's probably that we added them and they persist so we don't need to use these env variables.